deploy/openshift/test-app-template.yaml

```diff
@@ -11,6 +11,12 @@ parameters:
 - name: SERVICE_NAME
   description: The name to use for the service
   required: true
+- name: IMAGE_NAME
+  description: The name to use for the image
+  required: true
+- name: IMAGE_VERSION
+  description: The name to use for the image version
+  required: true
```

Comment:
these should not be required, if they are not set we should default to "kiali/kiali-test-service" and "latest"

Reply:
changed!